### ISSUE TITLE
GFX 100S 16 bits white point

### ISSUE THREAD

**Issue by jas01** [darktable-org/rawspeed#354]
When importing a 16 bits RAW uncompress file from the GFX100S into darktable, they are a problem with the white level. Every image are way too bright.

To fix this problem someone give me a solution and it's to change in the cameras.xml
```xml
<Sensor black="63" white="16383"/>
```
to
```xml
 <Sensor black="63" white="65535"/>
```
But when looking in the cameras.xml that would also affect 14bits file. 

So is they are any way to differentiate the GFX 100S 16 bits with the 14 bits ?, so darktable pickup the right white point

**Comment by kmilos** [comment]
See also https://github.com/darktable-org/rawspeed/issues/228

**Comment by dllu** [comment]
I can confirm that this fix is needed even after checking out the latest `develop` branch of `rawspeed` and compiling Darktable.

**Comment by LebedevRI** [comment]
I believe this was fixed by #421.
